### PR TITLE
disabled color

### PR DESCRIPTION
**Pull request by kanjanaSi**

### CODE REVIEW

**Comment by k0nserv** [comment]
Hey @kanjanaSi this looks really neat :) Can you post some screenshots of this as well?

File: Sources/SkyFloatingLabelTextField.swift [hunk 1]
```diff
@@ -74,21 +74,23 @@ open class SkyFloatingLabelTextField: UITextField { // swiftlint:disable:this ty
...
-            #endif
+        guard let placeholder = placeholder, let font = placeholderFont ?? font else {
+            return
         }
+        let color = isEnabled ? placeholderColor : disabledColor
+        #if swift(>=4.0)
+            attributedPlaceholder = NSAttributedString(
+            string: placeholder,
+            attributes: [
+            NSAttributedStringKey.foregroundColor: color, NSAttributedStringKey.font: font
```
Comment (k0nserv) [inline Sources/SkyFloatingLabelTextField.swift]: Please cleanup this indentation to match what it was before.

Comment (kanjanaSi) [reply Sources/SkyFloatingLabelTextField.swift]: Fixed!

**Comment by kanjanaSi** [comment]
@k0nserv 
screen shot 2017-10-26 at 7 52 23 pm

**Comment by k0nserv** [comment]
Awesome :) Would you be up for updating the README.md with documentation and the example project?

**Comment by k0nserv** [comment]
This control is confusing

image

It seems to be doing the opposite for me i.e when it's "disabled" the text fields is actually enabled and vice versa

**Comment by zarzonis** [comment]
@k0nserv Hi Hugo. Do you have any plan to release this? I was ready to implement this feature and I just found out this PR.

**Comment by k0nserv** [comment]
Hey @zarzonis yeah I'll do that right away :)

**Comment by k0nserv** [comment]
That's it released. Thanks a bunch @kanjanaSi for the awesome contribution 🎉

**Comment by kanjanaSi** [comment]
🎉🎉🎉🎉🎉

**Pull request merged**